src/import.js

```diff
@@ -96,6 +96,9 @@ Sk.doOneTimeInitialization = function (canSuspend) {
         if ((func.prototype instanceof Sk.builtin.object ||
              func === Sk.builtin.object) && !func.sk$abstract) {
             setUpClass(func);
+            func.tp$setattr = function(obj, pyName, value) {
```

Comment:
Move this to `setUpClass()`?

Reply:
Also, that's the wrong call signature for `tp$setattr`!
```
tp$setattr = function(pyName, data, canSuspend)
```

Reply:
moved and done